### PR TITLE
SidebarItem data cy

### PR DESCRIPTION
**Pull request by GeorgiSTodorov**
This attempts to fix two problems:
- Ugly border in Firefox for `SidebarItem`s with url-s.
### Before
Screenshot from 2020-03-12 14-10-13

- No data-cy for `SidebarItem`s.
### Before
Screenshot from 2020-03-12 14-11-55

### After
Screenshot from 2020-03-12 14-08-23

A third problem that I could not fix was that clicking on the left or right edge of a `SidebarItem` with url does nothing since it's the padding of the parent `<li>` element.
Peek 2020-03-12 13-50

### CODE REVIEW

**Comment by pierrebeitz** [comment]
updated test again to make CI pass. i think we should not have 4 commits here. 1 or 2 should be enough :)

**Comment by GeorgiSTodorov** [comment]
Thanks, @pierrebeitz .

**Comment by GeorgiSTodorov** [comment]
```
ERR! docker container or on a CI, you need to pass the '--ci' flag to prevent opening a

ERR! browser by default.
```
This is weird. Has anyone seen this before?

File: packages/shared/styles/styleUtils/resets/linkReset.ts [hunk 1]
```diff
@@ -7,4 +7,8 @@ export const linkReset = css`
   &:visited {
     color: inherit;
   }
+
+  :focus {
```
Comment (mperrotti) [inline packages/shared/styles/styleUtils/resets/linkReset.ts]: I'm strongly opposed to removing the focus outline on links. We should only hide it in specific cases where we have our own focus styles.

I'll push up a commit with my suggestion for how we should handle this for the sidebar items

**Comment by GeorgiSTodorov** [comment]
Thanks, @mperrotti .

**Pull request merged**